smack-openpgp/src/main/java/org/jivesoftware/smackx/ox/OpenPgpManager.java

```diff
@@ -566,7 +592,7 @@ public void run() {
                 if (contentElement instanceof CryptElement) {
                     for (CryptElementReceivedListener l : cryptElementReceivedListeners) {
                         l.cryptElementReceived(contact, message, (CryptElement) contentElement,
-                                        decrypted.getMetadata());
+                                decrypted.getMetadata());
```

Comment:
Unrelated whitespace change

Reply:
Are you happy if I move this change to another commit?

Reply:
Moved into separate commit